src/components/ColorPicker/ColorPicker.scss

```diff
@@ -56,6 +56,10 @@
       }
     }
   }
+
+  .color-picker__color-option {
+    background-color: var(--accent-color--400);
```

Comment:
here and in a lot of other places you opted for using the specific color number over `--accent-color--light` and `--accent-color--dark`. is there any specific reason or did you not know about this? ^^

Reply:
I did that because the design was switching a lot between specific color numbers :)

Reply:
I see, then looks good to me all in all 😌